en/flight_modes/altitude_mc.md

```diff
@@ -0,0 +1,42 @@
+# Altitude Flight Mode (Multicopter)
+
+[<img src="../../assets/site/difficulty_2.svg" title="Difficulty (Easy)" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;
+
+*Altitude mode* is a *relatively* easy-to-fly RC mode in which roll and pitch sticks control vehicle movement in the left-right and forward-back directions (relative to the "front" of the vehicle), yaw stick controls rate of rotation over the horizontal plane, and throttle controls speed of ascent-descent.
+
+When the sticks are released/centered the vehicle will level and maintain the current *altitude*. If moving in the horizontal plane the vehicle will continue until any momentum is dissipated by wind resistance. If the wind blows the aircraft will drift in the direction of the wind.
+
+> **Tip** *Attitude mode* is the safest non-GPS manual mode for new fliers. It is just like [Manual/Stabilized](../flight_modes/manual_stabilized_mc.md) mode but additionally stabilizes the vehicle altitude when the sticks are released.

```

Comment:
> Altitude is stabilized even with stick input, so when the sticks are released does not need to be added.

I don't understand. If I have an "outside deadzone" stick position the altitude is changing, so how can it be stabilised? Or to put it another way, isn't the point of this mode that it will lock altitude when the sticks are released?

Or even another way "what do you mean?"

Reply:
This depends on how you understand stabilised. To me, some entity can be stabilized even when it is changing. In this case the rate of change for the altitude is stabilized by the controller - meaning the velocity in z direction (which is kept constant for any given throttle stick position).
I think in this case you mean the vehicle is stabilized to a fixed/constant altitude. It's a minor detail, and I'm fine with the current wording.

Reply:
That is a very good point. WOuld you be happier with "locks: - e.g. :

> mode but additionally LOCKS to the current vehicle altitude when the sticks are released.

Reply:
Yes, 'locks' sounds good.

Reply:
Thanks. Fixed by https://github.com/PX4/px4_user_guide/commit/1cf90134106557a3ab8dac1144470140d844aac5